packages/create-react-native-library/templates/common/$package.json

```diff
@@ -53,7 +53,7 @@
     "@release-it/conventional-changelog": "^5.0.0",
     "@types/jest": "^28.1.2",
     "@types/react": "~17.0.21",
-    "@types/react-native": "0.68.0",
+    "@types/react-native": <%-/* TODO: bump version of rn to 0.69.0 */ project.newArchitecture ? '"0.69.5"':'"0.68.0"'-%>,
```

Comment:
Will adjust this todo after merge https://github.com/callstack/react-native-builder-bob/pull/271